src/backend/core/models.py

```diff
@@ -315,6 +316,7 @@ class Team(BaseModel):
     """
 
     name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=100, unique=True, null=False)
```

Comment:
Corner case: (which won't happen often)
Should they have equal max-length? 
does some characters slugify to a longer version?
ex: is `&` slugified to `and` ?

Reply:
Good question ! 

I tested it, to be sure. It seems that `&` is interpreted as a special caracter, and thus removed because I used `validate_slug` and not `validate_unicode_slug` ([see django docs](https://docs.djangoproject.com/en/4.2/ref/validators/#django.core.validators.validate_slug)).

I don't see another scenario where len(slug) > len(name).